k8s-specifications/postgres-deploy.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         app: demo-voting-app
     spec:
       containers:
-        - name: postgres
+        - name: postgres:12
```

Comment:
Probably you meant to add version to "image":
- name: postrges
  image: postgres:12